test/invoice_item_repository_test.rb

```diff
@@ -1,10 +1,12 @@
 require_relative '../test/test_helper'
 require_relative '../lib/invoice_item_repository'
+require 'bigdecimal/util'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.